### PR TITLE
Lukas

### PR DESCRIPTION
**Pull request by woocash2**

### CODE REVIEW

File: src/com/github/mtdrewski/GRAPH_moment/controller/DirectedEdgeLine.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        });
+        dy.addListener((observable, oldValue, newValue) -> {
+            rotate.setAngle(getAngle(newValue.doubleValue(), dx.doubleValue()));
+        });
+
+        triangle.layoutXProperty().bindBidirectional(endXProperty());
+        triangle.layoutYProperty().bindBidirectional(endYProperty());
+    }
+
+    @Override
```
Comment (natanjel) [inline src/com/github/mtdrewski/GRAPH_moment/controller/DirectedEdgeLine.java:33]: It doesn't override anything in edgeline or am I mistaken?

File: src/com/github/mtdrewski/GRAPH_moment/controller/GraphDrawerController.java [hunk 2]
```diff
@@ -29,6 +31,7 @@
     private boolean isUnsaved = false;
 
     private enum Mode {EDGE, SELECT, STANDARD}
+    protected boolean isDirected = false;
```
Comment (natanjel) [inline src/com/github/mtdrewski/GRAPH_moment/controller/GraphDrawerController.java:35]: shouldn't this information be in the graph.java? I think that there could be a moment where we would need this information from graph, but not from controller.

File: src/com/github/mtdrewski/GRAPH_moment/controller/DirectedEdgeLine.java (hunk 1 above)
Comment (woocash2) [reply src/com/github/mtdrewski/GRAPH_moment/controller/DirectedEdgeLine.java:33]: It does. Line 55 in EdgeLine

File: src/com/github/mtdrewski/GRAPH_moment/controller/GraphDrawerController.java (hunk 2 above)
Comment (woocash2) [reply src/com/github/mtdrewski/GRAPH_moment/controller/GraphDrawerController.java:35]: There are separate classes Graph and DirectedGraph, and storing a boolean inside one or each of them seems not elegant for me. I managed to make everything work without need of getting this info out of graph.java

**Pull request merged**